runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -852,6 +859,7 @@ private void toggleSidebar()
 		{
 			sidebarNavigationJButton.setIcon(new ImageIcon(sidebarClosedIcon));
 			sidebarNavigationJButton.setToolTipText("Close SideBar");
+			configManager.setConfiguration(CONFIG_GROUP, CONFIG_CLIENT_SIDEBAR_CLOSED, true);
```

Comment:
This is still backwards. When the tooltip text is `Close Sidebar` it means the sidebar is open, because the option on it is to *close* it. So sidebar closed shouldn't be true.